README.md

```diff
@@ -25,12 +25,106 @@
-* There's no reason to talk about Auto Layout or Size classes (yet). I've unchecked those boxes in the File Inspector in the Xcode project in this repo.
-* The Xcode project is ready to go in that the storyboard file is empty. As well, the Auto Layout and Size Classes are unchecked.
+Now that you know your way around Xcode (and specifically Interface Builder) a bit, it's time to take a closer look at how to lay out the graphical user interface of your application. As you do, we'll take a closer look at some of the concepts that come into play when you're working with _views_ in Xcode.
 
+First, open up the Xcode project provided in this repo. (It is at `LearningInterfaceBuilder/LearningInterfaceBuilder.xcodeproj`.) Then open up `Main.storyboard` in Interface Builder. Do you remember how to do that? If not, here's some help: Click once on `Main.storyboard` in the file listing at the left-hand side of the Xcode window. You should see the storyboard in Interface Builder.
 
+What do you see? An empty canvas! Right now, `Main.storyboard` is empty, as evidenced by the scene listing, which simply says **No Scenes**, as well as the fact that there is nothing on the _canvas_ (the area where you drag and drop view objects). That's about to change.
+
+### View Controllers
+
+The first thing you need to do is add a _view controller_ to the storyboard. As its name suggests, a view controller manages a single _view_. By "manage", we mean that it is responsible for coordinating how the view is drawn, and detecting any user input, such as touches or swipes. We'll talk about _views_ in a minute, but for now, let's get a view controller on that canvas.
+
+Do you remember where the _object library_ is? It's the pane at the bottom of the right-hand sidebar that contains a bunch of yellow icons. It looks like this:
+
+![Object Library](.images/xcode01.png)
+
+The object library contains all the different view objects—controllers, buttons, icons, and so forth—that can appear in your application. The top icon should be marked **View Controller**. That's what you want to add to your storyboard, so go ahead and drag that onto your canvas.
+
+![Drag a new view controller onto the canvas](.images/xcode02.png)
+
+You've just added a new view controller to your storyboard! This view controller will be responsible for managing the initial view of your application—everything the user sees when they first load your application.
+
+### Document Outline
+
+Notice that pane immediately to the left of Interface Builder's canvas, and to the right of the file listing? That's the _document outline_. When you first opened `Main.storyboard`, it was empty, and simply said **No Scenes**. Notice anything different? You should! Now that you have added a view controller, you'll see **View Controller Scene** in the document outline. There are _disclosure triangles_ next to that entry that show you all the objects associated with that scene.
+
```

Comment:
Love how you introduce this. Also, I couldn't come up with a good name for those things.. disclosure triangles.